specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimapis.json

```diff
@@ -4415,6 +4415,283 @@
         },
         "x-ms-odata": "./definitions.json#/definitions/TagResourceContract"
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/wikis/default": {
+      "head": {
+        "tags": [
+          "ApiWiki"
+        ],
+        "operationId": "ApiWiki_GetEntityTag",
+        "description": "Gets the entity state (Etag) version of the Wiki for an API specified by its identifier.",
+        "x-ms-examples": {
+          "ApiManagementHeadApiWiki": {
+            "$ref": "./examples/ApiManagementHeadApiWiki.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Operation completed successfully.",
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
```

Comment:
>./apimanagement.json#/definitions/ErrorResponse [](http://example.com/codeflow?start=23&length=47)

Please use common-types v3 for ErrorResponse type, instead of locally defined.

Reply:
This is not done at all places yet in other swagger files.